src/main/java/com/limechain/sync/warpsync/runtime/RuntimeVersion.java

```diff
@@ -0,0 +1,53 @@
+package com.limechain.sync.warpsync.runtime;
+
+import io.emeraldpay.polkaj.scale.ScaleCodecReader;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.math.BigInteger;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class RuntimeVersion {
+    String specName;
+    String implementation;
+    BigInteger authoringVersion;
+    BigInteger specVersion;
+    BigInteger implementationVersion;
+    BigInteger transactionVersion;
+    BigInteger stateVersion;
+    RuntimeApis runtimeApis;
```

Comment:
Almost none of the fields in this PR are `private`. Would be good to add that everywhere where it's missing unless we have a good reason to leave the fields package private (default).